packages/cfd/src/features/Containers/cfd-password-manager-modal/cfd-password-manager-tab-content.tsx

```diff
@@ -0,0 +1,194 @@
+import React from 'react';
+import { Tabs, DesktopWrapper, Div100vhContainer, MobileWrapper, ThemedScrollbars } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { isMobile, validLength, validPassword, getErrorMessages, getCFDPlatformLabel } from '@deriv/shared';
+import { useTradingPlatformInvestorPasswordChange, useTradingPlatformPasswordChange } from '@deriv/api';
+import { FormikErrors } from 'formik';
+import TradingPasswordManager from '../../../Containers/trading-password-manager';
+import InvestorPasswordManager from '../../../Containers/investor-password-manager';
+import {
+    TCFDPasswordManagerTabContent,
+    TFormValues,
+    TPasswordManagerModalFormValues,
+} from '../../../Containers/props.types';
+import { CFD_PLATFORMS, QUERY_STATUS, PASSWORD_TYPE } from '../../../Helpers/cfd-config';
+import { TStatus } from './cfd-password-manager-modal';
+
+export const CFDPasswordManagerTabContent = ({
+    toggleModal,
+    selected_login,
+    email,
+    setPasswordType,
+    multi_step_ref,
+    platform,
+    onChangeActiveTabIndex,
+    account_group,
+}: TCFDPasswordManagerTabContent) => {
+    const {
+        mutateAsync: changePassword,
+        status: change_password_status,
+        error: change_password_error,
+    } = useTradingPlatformPasswordChange();
+    const {
+        mutateAsync: changeInvestorPassword,
+        status: change_investor_password_status,
+        error: change_investor_password_error,
+    } = useTradingPlatformInvestorPasswordChange();
+    const [active_tab_index, setActiveTabIndex] = React.useState<number>(0);
+    const [error_message_investor, setErrorMessageInvestor] = React.useState<string>('');
+    const [is_submit_success_investor, setIsSubmitSuccessInvestor] = React.useState<boolean>(false);
+
+    // view height - margin top and bottom of modal - modal title - modal content margin top and bottom - table title
+    const container_height = 'calc(100vh - 84px - 5.6rem - 8.8rem - 4rem)';
+    const validatePassword = (values: TFormValues) => {
+        const errors: FormikErrors<TFormValues> = {};
+
+        if (
+            !validLength(values.new_password, {
+                min: 8,
+                max: 25,
+            })
+        ) {
+            errors.new_password = localize('You should enter {{min_number}}-{{max_number}} characters.', {
```

Comment:
what is the difference between doing this and doing `localize('You should enter 8-25 characters.')` 😅? doesn't look like there's text formatting for the `min_number` `max_number` and it looks hardcoded enough to just inline it